### ISSUE TITLE
Feature request: Support whitelisting of apps

### ISSUE THREAD

**Issue by pouldam** [jcbastosportela/logioptspp#10]
It would be nice if the app supported the option to use a whitelist instead of a blacklist.

I only have one additional application where I would like to add the smooth scrolling, thus a whitelist would make it a lot easier to configure.

One of the reasons why I do not want smooth scrolling enabled for all apps is that configuration of mouse buttons per app level goes down the drain as every app inherits the configuration for the Chrome browser.

**Comment by pouldam** [comment]
Perfect! - that sounds exactly like what I am seeking. However, when I try to launch v0.0.1 of your app I get an error message stating that ucrtbased.dll was not found. Closing the dialog I get the same fault just with VCRUNTIME140_1D.dll, MSVCP140D.dll and VCRUNTIME140D.dll.

I guess this is all related to v0.0.1 being a debug build?

Would you suggest I just seek out the debug libraries on the internet or do you have a release version in your drawer somewhere?

Thanks in advance.

Oh - and soooo cool to finally have smooth scrolling in more apps. I don't know why Logitech miss out on that opportunity.

**Comment by pouldam** [comment]
Ah, sorry - yeah I thought you were the author :-)

Anyway, I actually downloaded the v0.0.1 build from the release page and it was that one that gave me the errors (v0.0.2 runnning fine). I can see that the release notes for v.0.0.2 says that the build has been changed to a release build. Thus I assume that the v0.0.1 release is a debug build. That also matches the errors I am getting - the names of the missing DLLs all end on the letter 'd' which makes me assume that they are debug DLLs which my system currently lacks.

I guess the missing DLLs should be pretty easy to dig up from a reliable source (Microsoft), so I will just give that a shot.

Thanks for pointing me in the right direction!
